### DISCUSSION TITLE
pip packages for submission

### DISCUSSION THREAD

**Discussion by akashjinandra** [usnistgov/ARIAC#226] [Q&A]
Hello, currently it the docker submission it allows debian packages to be installed, what about pip packages? Currently the tf2 python doesn't work in ROS 2 Galactic

**Comment by 130s** [comment]
I see `pip` section in the submission doc after https://github.com/usnistgov/ARIAC/pull/227

**Comment by jaybrecht** [comment]
pip packages and custom build scripts were added as part of v1.4's automated evaluation
